spec/factories/project.rb

```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
   factory :project do
-    title 'Lorem ipsum project'
+    title 'Lorem ipsum'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.